src/App.js

```diff
@@ -1,12 +1,185 @@
-import React from 'react';
+//working: for one pokemon - :)
+
+// import React, {useEffect, useState} from 'react';
+// import './App.css';
+// import axios from "axios";
+// // import Button from "./assets/components/button";
+// import PokemonCard from "./assets/components/PokemonCard";
+// // import Hans from "./assets/components/Hans";
+//
+// function App() {
+//
+//     const [pokemonData, setPokemonData] = useState('');
+//     const [endPoint, setEndPoint] = useState('https://pokeapi.co/api/v2/pokemon/jigglypuff')
+//     const [counter, setCounter] = useState(0);
+//
+//
+//     useEffect(() => {
+//
+//         async function fetchData() {
+//             try {
+//                 const result = await axios.get(endPoint);
+//                 console.log(result.data);
+//                 setPokemonData(result.data)
+//             } catch (e) {
+//                 console.error(e);
+//             }
+//         }
+//
+//         fetchData()
+//         console.log("we gaan beginnen :)")
+//     }, [endPoint])
+//
+//
+//     return (
+//         <div>
+//             {/*<Hans/>*/}
+//             <PokemonCard endPoint={endPoint}/>
+//
+//             {/*<Button isDisabled={true} type="button"*/}
+//             {/*        clickHandler={() => setCounter(pokemonData.previous)}> vorige </Button>*/}
+//             {/*<Button isDisabled={false} type="button"*/}
+//             {/*        clickHandler={() => setCounter(pokemonData.next)}> volgende </Button>*/}
+//
+//         </div>
+//     );
+// }
+//
+// export default App;
+
+
+
+
+//working: for 20 pokemon - :)
+
+import axios from "axios";
 import './App.css';
+import React, {useState, useEffect} from "react";
+import PokemonCard from "./assets/components/PokemonCard";
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokemonData, setPokemonData] = useState('');
```

Comment:
Dit zal initieel van het type string zijn als je als default value `''` hanteert. Uiteindelijk wordt het een object dus dan zou ik als default value `undefined` gebruiken en als je het gebruikt eerst checkt dat het niet undefined is

Reply:
dit snap ik niet goed, bedoel je hier mee dat ik moest [ ] gebruiken ipv ' ' ?